pkgs/os-specific/linux/msi-ec/default.nix

```
@@ -0,0 +1,52 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  linuxPackages,
+  kernel ? linuxPackages.kernel,
+  ...
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```
https://nix.dev/guides/best-practices#recursive-attribute-set-rec

---

This is an unsubstantial nit. Though I also prefer avoiding `rec` (there is finalAttrs too, as a solution) there is no clear guideline on this.